apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector/preview.scss

```diff
@@ -0,0 +1,189 @@
+@import '~@wordpress/base-styles/variables';
+@import './shared.scss';
+
+// Preview positioning
+$preview-max-width: 960px;
+$preview-right-margin: 24px;
+$preview-left-sidebar-reduced: calc( 30% + #{$admin-sidebar-width-collapsed} );
+$preview-left-sidebar-full: calc( 30% + #{$admin-sidebar-width} );
```

Comment:
Will `$admin-sidebar-width` work both in wp-admin when WP sidebar is visible _and_ when Gutenberg is iframed in Calypso and the sidebar is hidden?

Reply:
idk :sweat_smile: -- But all I did in f4adda3 is remove two SASS var definitions with two other ones that I'm importing from `@wordpress/base-styles`. That shouldn't change any behavior, should it?

Reply:
>That shouldn't change any behavior, should it? 

It shouldn't indeed and sounds fair! :-) Just good to test both scenarios anyway always when CSS is touched for anything FSE plugin related.